plugins/pulp_rpm/plugins/importers/yum/listener.py

```diff
@@ -194,8 +195,14 @@ def download_succeeded(self, report):
             except (verification.VerificationException, util.InvalidChecksumType):
                 # verification failed, unit not added
                 return
+
+            # we need to read package header in order to extract signature info
+            # unit is not added if header cannot be read
+            headers = rpm_parse.package_headers(report.destination)
```

Comment:
Does this work for DRPMs? I assume yes since this listener gets used to sync DRPMs.